src/base/preferences.cpp

```diff
@@ -1910,6 +1914,86 @@ void Preferences::setMagnetLinkAssoc(bool set)
 }
 #endif
 
+#ifdef Q_OS_MAC
+namespace
+{
+    CFStringRef torrentExtension = CFSTR("torrent");
+    CFStringRef magnetUrlScheme = CFSTR("magnet");
+}
+
+bool Preferences::isTorrentFileAssocSet()
+{
+    bool isSet = false;
+
+    CFStringRef torrentId = UTTypeCreatePreferredIdentifierForTag(kUTTagClassFilenameExtension, torrentExtension, NULL);
+    if (torrentId != NULL)
+    {
```

Comment:
Fix coding style.
We don't use line break before opened brace in conditionals, loops and switches.